contrib/go.mongodb.org/mongo-driver.v2/mongo/orchestrion.yml

```diff
@@ -0,0 +1,21 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
If we are adding orchestrion support, we should add an integration test as well (under `internal/orchestrion/_integration`).